adapters/jwplayer/utils.go

```diff
@@ -76,6 +76,71 @@ func GetAppnexusExt(placementId string) json.RawMessage {
 	return jsonExt
 }
 
+type xandrVideoExt struct {
+	Appnexus xandrVideoExtParams `json:"appnexus"`
+}
+
+type xandrVideoExtParams struct {
+	Context xandrContext `json:"context,omitempty"`
+}
+
+type xandrContext int
+
+const (
+	Unknown   xandrContext = 0
+	PreRoll   xandrContext = 1
+	MidRoll   xandrContext = 2
+	PostRoll  xandrContext = 3
+	Outstream xandrContext = 4
+)
+
+func SetXandrVideoExt(video *openrtb2.Video) {
+	context := GetXandrContext(*video)
+	if context == Unknown {
+		return
+	}
+
+	videoExt := xandrVideoExt{
+		Appnexus: xandrVideoExtParams{
+			Context: context,
+		},
+	}
+	video.Ext, _ = json.Marshal(videoExt)
+}
+
+func GetXandrContext(video openrtb2.Video) xandrContext {
+	if IsOutstream(video.Placement) == true {
```

Comment:
1. don't need true
2. maybe: !IsInstream which is arguably a better representation based on the placement type definitions: https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list--placement-subtypes---video-

Reply:
I'm not sure I agree. !isInstream doesn't mean it's necessarily outstream. If the argument is null, it would be wrongfully considered outstream. 
My logic is: 
1) check for explicit outstream. 
2) check for a start delay 
2a) if none, we cannot determine; return unknown
2b) if explicit, determine the ad break offset